vue/src/components/loki/LogLabelValuesCont.vue

```diff
@@ -76,7 +76,7 @@ export default defineComponent({
       () => props.query,
       (query) => {
         labels.value.forEach((label) => {
-          label.selected = query?.includes(label.name) && query?.includes(label?.name)
+          label.selected = query?.includes(label.name) && query?.includes(label?.value)
```

Comment:
Nitpick: you don't need the optional chaining (`?`) here. Nor `query` nor `label` can be `null`. Could you check your VS Code settings?